### PR TITLE
Recommend newest jdk in docs

### PR DESCRIPTION
**Pull request by michaltomco**

### CODE REVIEW

File: optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc [hunk 1]
```diff
@@ -775,7 +775,7 @@ Your mileage may vary.
 * Operating System
 ** No experimentally proven advice yet (but prefer Linux anyway).
 * JDK
-** Version: Java 7 can be between 10% and 25% faster than Java 6. But Java 8 however is usually not significantly faster than Java 7.
+** Version: Our benchmarks have consistently shown improvements in performance when comparing new JDK releases with their predecessors. It is recommended to use the latest one.
```
Comment (triceo) [inline optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc:778]:
```suggestion
** Version: Our benchmarks have consistently shown improvements in performance when comparing new JDK releases with their predecessors. It is therefore recommended to use the latest available JDK.
```

Also, if you say things such as "consistently shown improvements", you may want to justify your claims with data. Perhaps a link to the blogs would be useful here?

**Review by triceo — CHANGES_REQUESTED** [review]

File: optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc (hunk 1 above)
Comment (michaltomco) [reply optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc:778]: If I put a link to the newest one here, wouldn't we be sort of obliged to update this doc with every new blog release? Maybe alternatively we can put a sentence there pointing people to the blog archives above?

Comment (michaltomco) [reply optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc]: Which one would you prefer?

option A: putting a link on the word "benchmarks" to the latest blog
option B: adding a sentence in the spirit of: "See our blog archives for more information on these comparisons."
option C: something else

Comment (triceo) [reply optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc]: B.

**Review by triceo — APPROVED** [review]

**Pull request merged**